.gemini.yml

```diff
@@ -0,0 +1,25 @@
+rootUrl: http://127.0.0.1:3000/test/visual
+gridUrl: http://127.0.0.1:4444/wd/hub
+screenshotsDir: ./test/visual/screens
+
+system:
+  plugins:
+    express:
+      port: 3000
+    sauce: true
```

Comment:
The reference images probably will not match to what you'll get locally so you can run "gemini update test/visual" first to update the images